lib/promise.js

```diff
@@ -25,21 +25,23 @@ function shimCLS(object, functionName, fnArgs){
 
 // Core
 shimCLS(Promise, 'join', [-1]);
-shimCLS(Promise.prototype, 'then', [0, 1, 2]);
-shimCLS(Promise.prototype, 'spread', [0, 1]);
-shimCLS(Promise.prototype, 'catch', [-1]);
-shimCLS(Promise.prototype, 'error', [0]);
+shimCLS(Promise.prototype, 'then',    [0, 1, 2]);
+shimCLS(Promise.prototype, 'spread',  [0, 1]);
+shimCLS(Promise.prototype, 'catch',   [-1]);
+shimCLS(Promise.prototype, 'error',   [0]);
 shimCLS(Promise.prototype, 'finally', [0]);
 
 // Collections
-shimCLS(Promise, 'map', [1]);
-shimCLS(Promise, 'reduce', [1]);
-shimCLS(Promise, 'filter', [1]);
-shimCLS(Promise, 'each', [1]);
-shimCLS(Promise.prototype, 'map', [0]);
-shimCLS(Promise.prototype, 'reduce', [0]);
-shimCLS(Promise.prototype, 'filter', [0]);
-shimCLS(Promise.prototype, 'each', [0]);
+shimCLS(Promise, 'map',       [1]);
+shimCLS(Promise, 'mapSeries', [1]);
```

Comment:
Only add the `mapSeries` changes, remove all the spaces you have added in other methods and shims.